neon_audio/tts/neon.py

```diff
@@ -130,18 +132,37 @@ def get_requested_tts_languages(msg) -> list:
 
 
 class NeonPlaybackThread(PlaybackThread):
-    def __init__(self, queue):
-        PlaybackThread.__init__(self, queue)
+    def __init__(self, queue, bus=None):
+        LOG.info("Initializing NeonPlaybackThread")
+        PlaybackThread.__init__(self, queue, bus=bus)
+
+    def begin_audio(self, message=None):
+        # TODO: Mark signals for deprecation
+        check_for_signal("isSpeaking")
+        create_signal("isSpeaking")
+        PlaybackThread.begin_audio(self, message)
+
+    def end_audio(self, listen, message=None):
+        PlaybackThread.end_audio(self, listen, message)
+        # TODO: Mark signals for deprecation
+        check_for_signal("isSpeaking")
 
     def _play(self):
         ident = self._now_playing[3]
+        if not ident and len(self._now_playing) >= 5 and \
```

Comment:
Backwards-compat here can be dropped without warning since queue is managed entirely in this package and ovos-audio